internal/pkg/term/prompt/select.go

```diff
@@ -190,3 +190,12 @@ func parseValueFromOptionFmt(formatted string) string {
 	}
 	return strings.TrimSpace(formatted)
 }
+
+func parseValuesFromOptions(formatted string) string {
```

Comment:
This works well as well with [`FriendlyText`](https://github.com/aws/copilot-cli/blob/f6940a431e98b2992bcb0a2fdbfde901580f4c9b/internal/pkg/term/prompt/select.go#L44) right? I think it does but I just wanted to double check.

We use `FriendlyText` in `copilot storage init` at the "lifecycle" prompt.